lib/cfg/cfg_Base.mli

```diff
@@ -1,9 +1,15 @@
+open Cfg_Node
```

Comment:
A comment for the whole file, in few lines you call the same things in three different ways: index, idx and id. This is confusing